tests/test_request.py

```diff
@@ -152,69 +152,34 @@ def test_clean_credit_card(self):
             {
                 "name": "deprecated last_4_digits is cleaned to last_digits",
                 "input": {
-                    "credit_card": {
-                        "issuer_id_number": "123456",
-                        "last_4_digits": "1234",
-                    },
-                    "device": {"ip_address": "1.1.1.1"},
+                    "issuer_id_number": "123456",
+                    "last_4_digits": "1234",
                 },
                 "expected": {
-                    "credit_card": {
-                        "issuer_id_number": "123456",
-                        "last_digits": "1234",
-                    },
-                    "device": {"ip_address": "1.1.1.1"},
+                    "issuer_id_number": "123456",
+                    "last_digits": "1234",
                 },
             },
             {
                 "name": "6 digit iin, 4 digit last_digits",
                 "input": {
-                    "credit_card": {
-                        "issuer_id_number": "123456",
-                        "last_digits": "1234",
-                    },
-                    "device": {"ip_address": "1.1.1.1"},
+                    "issuer_id_number": "123456",
+                    "last_digits": "1234",
                 },
                 "expected": {
-                    "credit_card": {
-                        "issuer_id_number": "123456",
-                        "last_digits": "1234",
-                    },
-                    "device": {"ip_address": "1.1.1.1"},
+                    "issuer_id_number": "123456",
+                    "last_digits": "1234",
                 },
             },
             {
                 "name": "8 digit iin, 2 digit last_digits",
                 "input": {
-                    "credit_card": {
-                        "issuer_id_number": "12345678",
-                        "last_digits": "34",
-                    },
-                    "device": {"ip_address": "1.1.1.1"},
+                    "issuer_id_number": "12345678",
+                    "last_digits": "34",
                 },
                 "expected": {
-                    "credit_card": {
-                        "issuer_id_number": "12345678",
-                        "last_digits": "34",
-                    },
-                    "device": {"ip_address": "1.1.1.1"},
-                },
-            },
-            {
-                "name": "8 digit iin, 4 digit last_digits",
```

Comment:
Removed this test since the cleaning doesn't contain the validation logic that would otherwise would disallow "8 digit iin, 4 digit last_digits"